docs/milo/manual/chapters/110_pre_flight_checks.md

```diff
@@ -48,7 +49,7 @@ If you are not happy with the amount of backlash then you can snug up the Anti-b
 
 Make sure you tram your spindle appropriately to your work table.
 
-If you are using a fixture of some kind, tram your spindle with reference to that instead.
+If you are using a fixture, of some kind, tram your spindle to that instead.
```

Comment:
Not a fan of this change with the additional text, I think it reads neater without the extra comma.

```suggestion
If you are using a fixture, tram your spindle to that instead.
```